test/e2e/tests/request-queuing/ui.spec.js

```diff
@@ -124,10 +124,16 @@ async function confirmTransaction(driver) {
 }
 
 async function switchToNetworkByName(driver, networkName) {
-  await driver.clickElement('[data-testid="network-display"]');
+  await driver.clickElement('.mm-picker-network');
```

Comment:
any reason why we have this change ?

Reply:
Because within `popup.html` at this small viewport, the tiny/condensed network picker displays instead of the full network picker that shows the complete name.

<img width="180" alt="SCR-20240719-ijht" src="https://github.com/user-attachments/assets/b2779522-2b59-4fe0-9f50-269631f74404">